using-conjoin/using-coinjoin.md

```diff
@@ -1,49 +1,27 @@
 # Using CoinJoin
+To use Wasabi Wallet for private payments, follow these steps:
 
-1. Download Wasabi Wallet. Follow the detailed instructions [<mark style="color:red;">**here**</mark>](https://app.gitbook.com/o/vkgJ5Qo6ByYsQmJoONIK/s/PdwveKS34LFQnTOWXGHS/)<mark style="color:red;">**.**</mark>
-2. Make sure that you have at least a balance of 0.11 bitcoin in your Wasabi wallet.
-   * To add bitcoin to your wallet, <mark style="color:red;">**check “Receiving Bitcoin” from the**</mark> [<mark style="color:red;">**Wasabi wallet guide**</mark>](https://app.gitbook.com/o/vkgJ5Qo6ByYsQmJoONIK/s/PdwveKS34LFQnTOWXGHS/)<mark style="color:red;">**.**</mark>
-   * You cannot CoinJoin directly from your hardware wallet, you need to have funds in the wallet you create using Wasabi.
-3. Open your Wasabi Bitcoin wallet from the right-hand menu, under “Wallet Explorer”.
-4. Select “CoinJoin” from the dropdown menu.
-5. Select the coins you want to make private by adding a checkmark beside each one.
-   * You can select up to 7 coins.
-   * The bitcoin balance needs to have enough to meet the minimum balance requirements to participate in the CoinJoin round.
-   * The round will begin once the timer elapses.
-
-{% hint style="danger" %}
-### <mark style="color:red;">**WARNING**</mark>
-
-Only select coins that have the same anonymity set. Else, this will reduce the privacy of your coins. Make sure you only select coins that have either a red, yellow or green shield. The green shield with the check mark in it means that those coins are successfully private.&#x20;
-{% endhint %}
-
-5\. Make sure the anonymity set is set to 50 (Green shield with check mark) by clicking on the large shield icon under “Target” at the bottom of the page.
-
-6\. Type in your password.&#x20;
-
-7\. Click enqueue.&#x20;
-
-* You will only need to enter your password once. The CoinJoin process will restart itself automatically until the target anonymity set is reached.
-* The CoinJoin process will restart itself automatically until the target anonymity set is reached.&#x20;
-
-9\. The CoinJoin process begins either once the time elapses (1 hour) or 100 peers have registered themselves.
+1. Download and install Wasabi Wallet and follow the detailed instructions provided in [<mark style="color:red;">**this guide**</mark>](https://app.gitbook.com/o/vkgJ5Qo6ByYsQmJoONIK/s/PdwveKS34LFQnTOWXGHS/)<mark style="color:red;">**.**</mark>
+2. Open your wallet and make sure you have at least 5000 satoshis, as well as enough funds for mining fees for each coinjoin round you plan to participate in. If you need to add bitcoin to your wallet, refer to the "Receiving Bitcoin" section of the Wasabi Wallet guide. Please note that coinjoin cannot be done directly from a hardware wallet and requires funds to be in the Wasabi-created wallet.
```

Comment:
IMO we should just remove the section mentioning hardware wallet, because soon it will be possible with Trezor.

```suggestion
2. Open your wallet and make sure you have at least 5000 satoshis, as well as enough funds for mining + coordination fees. If you need to add bitcoin to your wallet, refer to the "Receiving Bitcoin" section of the Wasabi Wallet guide.
```